src/views/helpers/index.ts

```diff
@@ -164,7 +164,7 @@ export function defaultBandViewProperties() {
     },
     hoverDimension: 'auto' as LineHoverDimension,
     upperColumn: '',
-    mainColumn: DEFAULT_AGGREGATE_FUNCTION,
```

Comment:
did we abandon this in the end? Not sure if the default_aggregate is still a relevant thing in general

Reply:
`DEFAULT_AGGREGATE_FUNCTION` is still used elsewhere, so that's why i didn't delete the constant. But we do not need it in the places where i changed it.

Reply:
Totally got that. Just wondering from a product standpoint if it's still something we wanted to do. @barbaranelson did we end up nixing this for all usages / should we?